tensorrt_llm/commands/serve.py

```diff
@@ -637,7 +725,8 @@ def resolve_command(self, ctx, args):
     commands={
         "serve": serve,
         "disaggregated": disaggregated,
-        "disaggregated_mpi_worker": disaggregated_mpi_worker
+        "disaggregated_mpi_worker": disaggregated_mpi_worker,
+        "mmemb_serve": serve_encoder
```

Comment:
Should we use a more verbose name: `mm_embedding_serve`?

Reply:
makes sense.